packages/@core.keystone/test.utils.js

```diff
@@ -26,7 +30,15 @@ const TESTS_REAL_CLIENT_REMOTE_API_URL = conf.TESTS_REAL_CLIENT_REMOTE_API_URL |
 const { SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION } = require('@condo/domains/user/gql.js')
 const http = require('http')
 const https = require('https')
-const { flattenDeep, fromPairs, toPairs } = require('lodash')
+const { flattenDeep, fromPairs, toPairs, set } = require('lodash')
+const fs = require('fs')
+const { BILLING_RECEIPT_SERVICE_FIELD_NAME } = require('@condo/domains/billing/constants/constants')
```

Comment:
you shouldn't use anything from apps inside the package

Reply:
This line was removed.